### ISSUE TITLE
Sending mail using nodemailer is slower

### ISSUE THREAD

**Issue by mohamedmohsins** [nodemailer/nodemailer#1626]
@andris9 @dfellis 
In Node JS sending Gmail's email using the npm nodemailer with an attachment of 10 MB takes 76 secs but the same mail directly sent from Gmail takes 18 sec.

Nodemailer version: 6.4.6

SMTP Configuration : 

{
	host: 'smtp.gmail.com',
	port: 587,
	tls: { rejectUnauthorized: false, minVersion: 'TLSv1' },
	secure: false,
	connectionTimeout: 30000,
	greetingTimeout: 30000,
	auth: {
		type: 'OAuth2',
		user: 'xxxx',
		clientId: 'xxxxxxx',
		clientSecret: 'xxxxxxx',
		refreshToken: 'xxxxxxxx'
	},
	requireTLS: true
}

**Comment by dfellis** [comment]
@mohamedmohsins while Github does still list me as the second biggest contributor, that was over 12 years ago

```sh
damocles@elack:~/oss/nodemailer(master)$ git log | grep -A 2 "David Ellis" | head -n 2
Author: David Ellis <d.f.ellis@ieee.org>
Date:   Fri Aug 12 16:55:39 2011 -0400
```

and none of my contributions exist anymore (and my total commit count doesn't even reach 1% of the commits in this repository)

```sh
damocles@elack:~/oss/nodemailer(master)$ git-fame 2>/dev/null | sed -n "/\(Author\|David Ellis\)/p"
| Author                |    loc |   coms |   fils |  distribution   |
| David Ellis           |      0 |      6 |      0 | 0.0/ 0.6/ 0.0   |
```

so including me in this issue doesn't really make much sense.

However, there are a multitude of reasons why sending email via this library will be slower than from Gmail itself, including things like:

1. Google possibly trusting requests from its own servers more implicitly and bypassing spam filters if sent via the web UI.
2. Mail sent via API may be put into a different queue than the Web UI, perhaps as a way to encourage upgrading your email account to an enterprise one?
3. The web UI may be communicating to a nearby web server to reduce the latency for transferring your 10MB attachment, while the SMTP server may only have a location in California and therefore reduce the possible bandwidth for the data transfer?

Basically, I doubt that this is nodemailer's fault.

**Comment by LeonMueller-OneAndOnly** [comment]
Last commit was 12 years ago, and still you provided a great, educational answer today. Thats what i call dedication to a project. Thank you @dfellis!

**Comment by andris9** [comment]
No reason to keep this issue open. Thank you @dfellis for answering it.
